snafu/log_generator_wrapper/trigger_log_generator.py

```diff
@@ -276,15 +282,17 @@ def emit_actions(self):
                     messages_received = self._check_es(int(start_time), int(end_time))
                 elif self.kafka_bootstrap_server:
                     messages_received = self._check_kafka(start_time, end_time + self.timeout)
-                    logger.info(
-                        "Current messages received is {}, waiting more time for kafka".format(
-                            messages_received
-                        )
-                    )
                 if messages_received == message_count:
                     received_all_messages = True
                 else:
                     logger.info("Message check failed. Retrying until timeout")
+                    if self.kafka_bootstrap_server:
```

Comment:
This if is redundant with the if at line 274. It wouldn't fall into here if the bootstrap server was undefined

Reply:
It is not, I only want to execute L290-L295 for kafka. Line 274 is an OR condition, so we enter into it for any condition that meets it (ES, cloudwatch or kafka), however I now only want to check for kafka.

Reply:
Ahhhhhhhhh ok. nevermind.